addOns/diff/src/main/javahelp/org/zaproxy/zap/extension/diff/resources/help/contents/diff.html

```diff
@@ -9,10 +9,17 @@
 <BODY>
 <H1>Diff</H1>
 <p>
-	The Diff add-on allows you to compare 2 requests or responses.
+	The Diff add-on allows you to compare 2 messages (requests or responses).
 </p>
 <p>
-	It is typically accessed via right click menu items.
+	It can be accessed by selecting two messages from any view component that shows more than one message e.g. the Sites tree and then opening the context menu (usually done by pressing the right-click button). You'll see the options "Compare 2 Requests", "Compare 2 Responses" there clicking one of it will open the diff window. 
+
+	Here's the screenshot of how the context menu will look like when you select two messages
+	<img src="../../common/images/context-menu.png"/>
+	Note: These two options are greyed out (disabled) when you select only one message and open the context menu.
+
+	The diff window shows the difference between two messages highlighted yellow.
```

Comment:
Add br

Reply:
Still needs the br.

Reply:
Actually it's missing the opening paragraph before.

Reply:
I have closed the paragraph tag; I dont think so that adding br is the solution, the document looks fine now.

Reply:
Right, the br was before noticing it was missing the opening tag.